v63002/u_inref/gtm8718.csh

```diff
@@ -0,0 +1,28 @@
+#!/usr/local/bin/tcsh -f
+#################################################################
+#								#
+# Copyright (c) 2018 YottaDB LLC. and/or its subsidiaries.	#
+# All rights reserved.						#
+#								#
+#	This source code contains the intellectual property	#
+#	of its copyright holder(s), and is made available	#
+#	under a license.  If you do not know the terms of	#
+#	the license, please stop and do not read further.	#
+#								#
+#################################################################
+#
+#
+#
+setenv gtm_test_autorelink_always 1
```

Comment:
What is the purpose of this line? Seems unnecessary.

Reply:
In the release note, it states if autorelink is in use, this ensures auto relink is in use

Reply:
gtm_test_autorelink_always is a test system environment variable that is randomly chosen by the test framework at subtest startup. $zroutines is set with or without auto-relink based on the random choice before control reaches gtm8718.csh. So setting this env var does not help. You can try running this test with V63001A (which does not have the code fix) multiple times and you will see only 50% of the times it will core (i.e. when the random choice was auto-relink). It is okay for gtm8718.csh to run with the random choice as that improves test coverage and since the test output is the same in both cases.

Reply:
As discussed, this line needs to be removed. Otherwise the changes are good to go.